data/tutorials/guides/0tt_05_s_expressions.md

```diff
@@ -0,0 +1,270 @@
+---
+id: s-expressions
+title: S-Expressions
+description: >
+  A tutorial on how to work with S-Expressions in OCaml
+category: "Tutorials"
+---
+
+# S-Expressions
+
+## Introduction
+
+S-expressions are a popular text serialisation format among the OCaml community. In the same way that JSON or YAML can be used to **pass structured data** between various components in a web-based context, S-expressions often fill the same role within an OCaml-to-OCaml context. The S-expression format isn't limited to OCaml and is quite old, as it was invented for the Lisp language.
+
+The [`sexplib`](https://github.com/janestreet/sexplib), created by Jane Street, provides support for S-expressions in OCaml. It was developped to provide support for S-expression to [Base](https://github.com/janestreet/base) and [Core](https://github.com/janestreet/core). However, S-expressions serialisation does not depend on these packages. In this tutorial, we only use `sexplib`. You can install it using opam.
```

Comment:
Do you want an explanation of what `sexplib` stands for and/or how to pronounce it?

Also, please check that my addition to the below code works there or if `opam list sexplib` needs to be somewhere else.

Reply:
@cuihtlauac ^^ Not sure you saw this, and we forgot to go over it in our check-in today.